### DISCUSSION TITLE
export custom fields in Excel

### DISCUSSION THREAD

**Discussion by BeckeBauer** [kimai/kimai#3365] [Questions & Answers]
In the Kimai plugins demo, I defined a custom field for invoices. After creation of an invoice, I click on the invoice and enter information into the custom field in the pop-up window. The data is shown in an additional column as defined by the plugin in the invoice list - so far, so good. 

However, when I export the invoice list, the information added to an invoice in the custom field cannot be found in excel. Is there anything which needs to be done/configured in order to make the custom field information appear after export?

**Comment by kevinpapst** [comment]
I tested (because I didn't knew myself) and ~~they do not get exported anywhere with this quick-download-export feature.
So you get them in Export module, but eg. not in Customer/Project/Activity and Invoice.
Please open a feature request here for that.~~ you are right, will be solved.

**Reply by kevinpapst** [reply]
Okay... I feel like Bart Simpson and need to repeat and say to myself:

> You shall not post before your first coffee.
> You shall not post before your first coffee.
> You shall not post before your first coffee.
> You shall not post before your first coffee.
> You shall not post before your first coffee.

This PR fixes the issue: https://github.com/kevinpapst/kimai2/pull/3366

**Reply by BeckeBauer** [reply]
😂😂😂
Thanx for fixing... I will test after merger...

**Reply by BeckeBauer** [reply]
It is working - custom field shows up after exporting - thank you 👍
